### ISSUE TITLE
v2 - Overriding class names not working

### ISSUE THREAD

**Issue by gpassarelli** [algolia/vue-instantsearch#517]
**Feedback on the v2 alpha version 🆕**

### Describe the bug / features
When you pass the prop `class-names` to  the components, as described in here [https://deploy-preview-416--vue-instantsearch.netlify.com/getting-started/styling.html#overriding-class-names](url)
 the classes are not beeing overriding, instead it add the attribute as `class-names="[object Object]" `

### Reproduction
https://codesandbox.io/s/l7jp6rxr3m
Inspect the element on you will see the attribute `class-names="[object Object]" `

### What is the version you are using?
v2.0.0-alpha.1

**Comment by Haroenv** [comment]
That's correct, thanks for notifying me of this, that guide was for v1 and it wasn't yet updated (as noted in the beginning of the guide). 

The feature however isn't present yet in v2. I'll add it 👍

**Comment by gpassarelli** [comment]
Haroen, thanks, I've noticed the note saying that the guide was not updated yet, I was just not sure if it was only the document that it was not updated, but the feature was present, or if the feature was not present yet;

I'm making some tests with the v2 branch, do you want me to keep opening issues for the things I find?

**Comment by Haroenv** [comment]
Yes, please do open issues! All feedback is welcome. 

Note by the way that the example you posted, you tried to replace the class names with the ones already used

**Comment by gpassarelli** [comment]
Alright, I'll.

I know, it's just because I copied from the template you guys provide and only updated the version. On my tests, I'm using my own class names.
